src/Operations.jl

```diff
@@ -586,12 +587,48 @@ function install_git(
     end
 end
 
+function temp_base_stdlibs_env(f::Function)
+    mktempdir() do dir
+        project_file = joinpath(dir, "Project.toml")
+        open(project_file, "w") do io
+            TOML.print(Dict(
+                "deps" => Dict(name => string(uuid) for (uuid, name) in last(last(Types.STDLIBS_BY_VERSION)))
+            ))
+        end
+        f(project_file)
+    end
+end
+
+function collect_artifacts(pkg_root::String; platform::AbstractPlatform=HostPlatform())
+    # Check to see if this package has an (Julia)Artifacts.toml
+    artifacts_tomls = Tuple{String,Dict}[]
+    for f in artifact_names
+        artifacts_toml = joinpath(pkg_root, f)
+        if isfile(artifacts_toml)
+            selector_path = joinpath(pkg_root, ".pkg", "select_artifacts.jl")
+
+            # If there is a dynamic artifact selector, run that in an appropriate sandbox to select artifacts
+            if isfile(selector_path)
+                temp_base_stdlibs_env() do project
```

Comment:
Instead of this, I think it is better to set `JULIA_LOAD_PATH` in the new process to `"@stdlib"`.

Reply:
(That is effectively what was used in the tests here anyway since your TOML.print went to stdout)

Reply:
That..... makes sense